applications/FluidDynamicsApplication/custom_strategies/schemes/simple_steady_adjoint_scheme.h

```diff
@@ -44,7 +44,7 @@ namespace Kratos
  * \f$\partial_{\mathbf{w}}J^{T}\f$ is returned by ResponseFunction::CalculateFirstDerivativesGradient.
  * \f$\mathbf{c}$ is the rotation and slip condition matrix
  */
-template <unsigned int TDim, class TSparseSpace, class TDenseSpace, unsigned int TBlockSize = TDim + 1>
+template <unsigned int TDim, class TSparseSpace, class TDenseSpace>
```

Comment:
I mean both the dimension and the block size templates. I don't think you need these as the schemes API doesn't involve bounded matrices. If you need the template for the auxiliaries used inside, you can check once for the `ProcessInfo` `DOMAIN_SIZE`.

Reply:
done :)